### ISSUE TITLE
Set up Angular

### ISSUE THREAD

**Issue by cfree** [denvercodeclub/denvercodeclub-com#8]
- [ ] `main.html`
- [ ] `Gulpfile.js`
- [ ] Bower
- [ ] `package.json`

**Comment by cfree** [comment]
I can't assign two people, but I want @BattyGrrl7 to help out here too. Pair programming?

**Comment by cfree** [comment]
(Don't start yet, this is contingent on another task. Sorry, I got excited)

**Comment by seansean11** [comment]
Too late, it's already merged :-)

**Comment by BattyGrrl7** [comment]
Sure!!!
